irc/nickserv.go

```diff
@@ -1055,10 +1055,10 @@ func nsSaregisterHandler(service *ircService, server *Server, client *Client, co
 		var failCode string
 		if err == errAccountAlreadyRegistered || err == errAccountAlreadyVerified {
 			errMsg = client.t("Account already exists")
-			failCode = "USERNAME_EXISTS"
+			failCode = "ACCOUNT_EXISTS"
 		} else if err == errNameReserved {
 			errMsg = client.t(err.Error())
-			failCode = "USERNAME_EXISTS"
+			failCode = "BAD_ACCOUNT_NAME"
```

Comment:
This should probably be `ACCOUNT_EXISTS` as well (it only triggers in rare Ergo configurations, but a typical trigger would be if nickname reservation is not strictly enforced and you're trying to reserve a nick that someone else has GROUP'ed)